eng/common/templates/steps/azure-public-roslyn-auth.yml

```diff
@@ -0,0 +1,6 @@
+steps:
```

Comment:
Everything under `eng/common` is owned by arcade, right? Is this code also being ported over there?

Reply:
I... don't know :) this is the "I got it working! Yay!" 

I'm sure someone will yell at me

Reply:
Definitely think we should move this out of common, unless and until its pushed down to arcade and we don't need it any more.